pkg/fatfs/fatfs_vfs/fatfs_vfs.c

```diff
@@ -35,6 +35,8 @@
 static int fatfs_err_to_errno(int32_t err);
 static void _fatfs_time_to_timespec(WORD fdate, WORD ftime, time_t *time);
 
+mtd_dev_t *fatfs_mtd_devs[FF_VOLUMES];
```

Comment:
Can this be static?

Reply:
No, it's also used by `pkg/fatfs/fatfs_diskio/mtd/mtd_diskio.c` and without VFS it has to be provided by the application.